docs/Guide/preconditions/command-cooldown.mdx

```diff
@@ -1,6 +1,278 @@
 ---
-sidebar_position: 0
+sidebar_position: 2
 title: Configuring command cooldowns
 ---
 
-## TODO
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Cooldowns are of vital importance for many bots to avoid spam, API ratelimits, etc. Luckily, Sapphire makes it easy to
+integrate them into your commands! At its simplest level, cooldowns can be used in specific commands via the
+[`cooldownDelay`][cooldowndelay] property in the command's options. This value is amount of milliseconds that a user will
+have to wait after using a command to use it again. Here's a basic example:
+
+<Tabs groupId="language-choice">
+<TabItem value="javascript" label="JavaScript" default>
+
+```javascript {7}
+const { Command } = require('@sapphire/framework');
+
+module.exports = class PingCommand extends Command {
+  constructor(context) {
+    super(context, {
+      // ...
+      cooldownDelay: 10_000 // 10_000 milliseconds
+    });
+  }
+};
+```
+
+</TabItem>
+<TabItem value="ESM" label="ESM">
+
+```javascript {7}
+import { Command } from '@sapphire/framework';
+
+export class PingCommand extends Command {
+  constructor(context) {
+    super(context, {
+      // ...
+      cooldownDelay: 10_000 // 10_000 milliseconds
+    });
+  }
+}
+```
+
+</TabItem>
+<TabItem value="typescript" label="TypeScript">
+
+```typescript {6}
+import { ApplyOptions } from '@sapphire/decorators';
+import { Command, CommandOptions } from '@sapphire/framework';
+
+@ApplyOptions<CommandOptions>({
+  // ...
+  cooldownDelay: 10_000 // 10_000 milliseconds
+})
+export class PingCommand extends Command {}
+```
+
+</TabItem>
+</Tabs>
+
+If you now try to run this command, and then run it again within 10 seconds, the command won't execute, and an error will
+be thrown. You can learn how to process that error [here][reporting-precondition-failure].
+
+:::info
+
+`cooldownDelay` only accepts a value of milliseconds, which is not known to be the easiest to read or calculate. To help,
+you can use [@sapphire/time-utilities], which includes utilities for time transformers.
+
+<Tabs groupId="language-choice">
```

Comment:
Ditto but is it even merged yet??

Reply:
She can't do that until #97 is merged and this will likely be merged sooner